x-pack/plugins/triggers_actions_ui/public/types.ts

```diff
@@ -518,13 +518,15 @@ export interface BulkActionsConfig {
   ) => void;
 }
 
+export interface RenderCustomActionsRowArgs {
```

Comment:
let me know if you come up with a better name please

Reply:
Nothing comes to my mind 🙈